alpine-compiler/Dockerfile

```
@@ -0,0 +1,9 @@
+FROM dddecaf/wsgi-base:alpine
+
+RUN set -eux \
+    # Install build tools. These can be required for the `pip-compile` step in
+    # child images that require to build packages from source.
+    && apk add g++ \
```

Just in case a dependency would need further build tools, I think the `build-base` package could be a safer bet than `g++`. Going with "don't optimize for an uncertain future" you can safely ignore my comment, though :wink: 

```suggestion
    && apk add build-base \
```

---

If that has a chance to save us from some random source build error later then it's so completely worth it